rotkehlchen/chain/evm/decoding/thegraph/decoder.py

```diff
@@ -102,57 +89,6 @@ def _get_user_address(self, address: ChecksumEvmAddress) -> ChecksumEvmAddress |
                 return None
         return address
 
-    def _decode_delegation_transferred_to_l2(self, context: DecoderContext) -> DecodingOutput:
-        """Decode a transfer delegating GRT from ethereum to arbitrum"""
-        user_address = self._get_user_address(hex_or_bytes_to_address(context.tx_log.topics[1]))
-        if not user_address or not self.base.is_tracked(user_address):
-            return DEFAULT_DECODING_OUTPUT
```

Comment:
you moved the removed code to ethereum right?

Reply:
Yes